daemon/cmd/daemon_main.go

```diff
@@ -1482,6 +1482,12 @@ func runDaemon() {
 		d.SendNotification(monitorAPI.AgentNotifyStart, repr)
 	}
 
+	// clean up all arp PERM entries that might have previously set by
+	// a Cilium instance
+	if !d.datapath.Node().NodeNeighDiscoveryEnabled() {
+		log.Info("Deleting old neighbor entries left by previous agent")
```

Comment:
We should log this only if we are going to delete entries (i.e. the stored link exists). OTOH, maybe it's enough to log that entries have been removed?